code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -82,7 +82,8 @@
 				L.visible_message("<span class ='danger'>[src] pounces on [L]!</span>", "<span class ='userdanger'>[src] pounces on you!</span>")
 				if(ishuman(L))
 					var/mob/living/carbon/human/H = L
-					H.apply_effect(10 SECONDS, WEAKEN, H.run_armor_check(null, MELEE))
+					H.apply_effect(10 SECONDS, KNOCKDOWN, H.run_armor_check(null, MELEE))
+					H.adjustStaminaLoss(30)
```

Comment:
I would probably increase this a little, to maybe 40? then you can pounce and disarm twice? seen as pouncing runs the risk of stunning yourself

Reply:
I kept the stats of pounce and disarm identical to mimic the original behavior, but I'd be fine giving pounce a little extra stamina damage if no one else thinks it'd be outside the scope of this PR.

Reply:
Aaaaand done.